### ISSUE TITLE
Show Team Permissions on Team site

### ISSUE THREAD

**Issue by rixx** [pretix/pretix#572]
When looking at a team site, I have to go to the edit mode to see the actual permissions associated with this team. This is especially annoying when I'm trying to compare permissions. A list of permissions granted to this team would go a long way to make this page more usable.

**Comment by raphaelm** [comment]
Very good point!

**Comment by parkcl** [comment]
Hi @raphaelm, I would like to look into this. Could you please assign it to me?

**Comment by raphaelm** [comment]
Hi @parkcl, sure, go ahead!

**Comment by raphaelm** [comment]
Hi @parkcl, everything going fine? :)

**Comment by aimanparvaiz** [comment]
@raphaelm can I take a shot at this?

**Comment by raphaelm** [comment]
@aimanparvaiz Sure!

**Comment by kika115** [comment]
it's free ??

**Comment by raphaelm** [comment]
@kika115 yes it is, sorry for the late reply, I was travelling. Are you still interested? Please go ahead and ask me if you have any questions!

**Comment by abramzog** [comment]
Is someone currently working on this issue?
If not I would like to try it.

**Comment by endersonmenezes** [comment]
Is someone currently working on this issue?
If not I would like to try it.

**Comment by raphaelm** [comment]
@endersonmenezes Feel free!

**Comment by raphaelm** [comment]
Yes

**Comment by ateuber** [comment]
This feature would be useful for us, too. Also when being in the event settings it would be useful to see which teams have access there. It happens frequently to us that someone creates an event with a new 1-person team and then goes on holiday.
